File: src/main/java/com/uwetrottmann/trakt5/entities/SyncErrors.java
package com.uwetrottmann.trakt5.entities;

import java.util.List;

public class SyncErrors {

    public List<SyncMovie> movies;
    public List<SyncShow> shows;
    public List<SyncSeason> seasons;
    public List<SyncEpisode> episodes;
    public List<SyncPerson> people;
    public List<Long> ids;

}

File: src/main/kotlin/com/redissi/trakt/entities/SyncErrors.kt
package com.redissi.trakt.entities

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
class SyncErrors(
    val movies: List<SyncMovie>? = null,
    val shows: List<SyncShow>? = null,
    val seasons: List<SyncSeason>? = null,
    val episodes: List<SyncEpisode>? = null,
    val people: List<SyncPerson>? = null,
    val ids: List<Long>? = null
)